infra/examples-dev/aws/main.tf

```diff
@@ -94,6 +94,19 @@ module "psoxy" {
   bulk_input_expiration_days     = var.bulk_input_expiration_days
 }
 
+module "connection_via_tenant_api" {
+  # dev example path: "../../../../terraform-aws-worklytics/examples/create_psoxy_connections"
+  # TODO URL of the actual repo to illustrate... initial version not released yet
+  source = "git::https://github.com/worklytics/terraform-aws-worklytics/examples/create_psoxy_connections?ref=v0.1.0"
+
+  # TODO review this: using the 1st unique ID while testing is OK, but maybe we should define a
+  #  an explicit variable for it
```

Comment:
Probably; and that variable by default could have `caller_gcp_service_account_ids[0]`